### ISSUE TITLE
Midi keyboard support

### ISSUE THREAD

**Issue by mrlem** [Achaaab/Bragi#8]
It would be really cool if Bragi could include a midi keyboard block, for physical keyboards.
If you need one for testing (after confinment), I've got a simple one!

**Comment by mrlem** [comment]
Also, I'm guessing integration through javax.sound.midi shouldn't be too hard.

**Comment by Achaaab** [comment]
Indeed, using computer keyboard is a hell and layout is still hard-coded.
I'd be happy to borrow your midi keyboard and try to integrate it.

**Comment by mrlem** [comment]
Tested the javax.sound API for my week-end github project. Works like a charm with the midi keyboard :smile:
